analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SecurityPInvokeMethodShouldNotBeCalled.CSharp11.cs

```diff
@@ -85,4 +85,27 @@ public void Somemethod()
             var hres2 = CoInitializeSecurity(0, -1, 0, 0, RpcAuthnLevel.None, RpcImpLevel.Impersonate, 0, EoAuthnCap.None, 0);
         }
     }
+
+    public unsafe partial class LibraryImportAttributeImports
+    {
+
+        // CoSetProxyBlanket can not be converted to LibraryImport because of the [MarshalAs(UnmanagedType.IUnknown)] attribute: https://github.com/dotnet/runtime/blob/main/docs/design/libraries/LibraryImportGenerator/Compatibility.md
```

Comment:
is it better to use a net7 github tag in the link? because the `main` doc path may change in the future